feat(blog): Hacktoberfest 2024 aftermath (#7681)
jenkins-infra/jenkins.io: content/blog/2024/11/13/2024-11-13-hacktoberfest.adoc

---
layout: post
title: "Jenkins' Participation in Hacktoberfest 2024"
tags:
- hacktoberfest
- event
- community
- newcomer
- outreach-programs
author: gounthar
links:
  discourse: https://community.jenkins.io/t/hacktoberfest-2024/20812
opengraph:
  image: /images/hacktoberfest/hacktoberfest_2024_social.jpg
---

image:/images/hacktoberfest/hacktoberfest_2024_logo.svg[Hacktoberfest,role=center,width='75%']

The Jenkins community is pleased to have participated in this year’s Hacktoberfest, an event focused on fostering open-source contributions and welcoming new contributors to our ecosystem.

As Hacktoberfest 2024 has come to a close, we're happy to reflect on the community's participation:

== Jenkins’ Contribution to Hacktoberfest

> This year, we saw an increase in community engagement, with contributions including *1,174 pull requests* across the `jenkinsci`, `jenkins-infra`, and `jenkins-docs` GitHub organizations. Notably, *243 pull requests* were part of the Hacktoberfest initiative, contributed by *67 unique participants*.

Reviewer: ```suggestion
This year, we saw an increase in community engagement, with contributions including *1,174 pull requests* across the `jenkinsci`, `jenkins-infra`, and `jenkins-docs` GitHub organizations.
Notably, *243 pull requests* were part of the Hacktoberfest initiative, contributed by *67 unique participants*.
```

Out of these submissions, *223 pull requests were successfully merged*, demonstrating the collaborative efforts and commitment of our contributors and maintainers alike, achieving an impressive 92% acceptance rate.

== Encouraging Newcomers

> One of our primary objectives during Hacktoberfest is to welcome and support new open-source contributors. The Jenkins community provided resources and guidance, assisting newcomers in understanding the procedures and standards for quality contributions.

Reviewer: ```suggestion
One of our primary objectives during Hacktoberfest is to welcome and support new open-source contributors.
The Jenkins community provided resources and guidance, assisting newcomers in understanding the procedures and standards for quality contributions.
```

> We are proud of all our participants, with special recognition for our seasoned contributors who continue to provide valuable input year after year. Notable contributions from some of our well-established members include:

Reviewer: ```suggestion
We are proud of all our participants, with special recognition for our seasoned contributors who continue to provide valuable input year after year.
Notable contributions from some of our well-established members include:
```

- author:jonesbusy[*jonesbusy*], submitting *41 validated pull requests*
- author:basil[*basil*], contributing *14 pull requests*
- author:kmartens27[*kmartens27*], adding *11 pull requests*

> Each of these individuals has consistently demonstrated their commitment to enhancing the Jenkins project, contributing their expertise to not only advance our goals but also mentor and support new contributors along the way.

Reviewer: ```suggestion
Each of these individuals has consistently demonstrated their commitment to enhancing the Jenkins project, contributing their expertise to advance our goals, and mentoring and supporting new contributors along the way.
```
Suggestion to adjust grammar.

> Each of these individuals has consistently demonstrated their commitment to enhancing the Jenkins projects, contributing their expertise to not only advance our goals but also mentor and support new contributors along the way.

Reviewer: ```suggestion
Each of these individuals has consistently demonstrated their commitment to enhancing the Jenkins project, contributing their expertise to not only advance our goals but also mentor and support new contributors along the way.
```

== Supporting Our Maintainers and Community

> The proactive involvement of our project maintainers has been instrumental in the success of Hacktoberfest this year. From preparing repositories by adding relevant topics and tags to ensuring clear, accessible contribution guidelines, their dedication has been crucial. Our heartfelt thanks go out to all maintainers for their role in fostering a supportive environment for our contributors.

Reviewer: ```suggestion
The proactive involvement of our project maintainers has been instrumental in the success of Hacktoberfest this year.
Their dedication to preparing repositories by adding relevant topics and tags to ensure clear, accessible contribution guidelines has been crucial.
Our heartfelt thanks go out to all maintainers for fostering a supportive environment for our contributors.
```
Suggestion to adjust the grammar and sentence structure.

== Looking Forward to Hacktoberfest 2025

> With an eye towards next year’s Hacktoberfest, we are keen to build on our success and continue improving our engagement with both long-standing community members and those new to the world of open-source. We hope to keep enriching our processes and support frameworks.

Reviewer: ```suggestion
With an eye towards next year’s Hacktoberfest, we are keen to build on our success and continue improving our engagement with both long-standing community members and those new to the world of open source.
We hope to keep enriching our processes and support frameworks.
```

The Jenkins community is proud of what we've achieved together and looks forward to welcoming more individuals into our ongoing open-source projects.

> If you are interested in contributing or simply learning more about our activities, we encourage you to engage with us.

Reviewer: ```suggestion
If you are interested in link:https://www.jenkins.io/participate/[contributing] or learning more about our activities, we encourage you to engage with us.
```
Slight word adjustment and add link to contribute page.

> Join our discussions and help shape the future of Jenkins! link:https://community.jenkins.io/t/hacktoberfest-2024/20812[]

Reviewer: ```suggestion
link:https://community.jenkins.io/t/hacktoberfest-2024/20812[Join our discussions] and help shape the future of Jenkins!
```
Suggestion to move link placement so it is part of the text.
